src/store/modules/layers.store.js

```diff
@@ -110,6 +110,14 @@ const getters = {
     getActiveLayerById: (state) => (geoAdminLayerId) =>
         state.activeLayers.find((layer) => layer.getID() === geoAdminLayerId),
 
+    /**
+     * Get layers with time config
+     *
+     * @returns {GeoAdminLayer[]} List of layers with time config
+     */
+    visibleLayersWithTimeConfig: (state, getters) =>
+        getters.visibleLayers.filter((layer) => layer.timeConfig.timeEntries.length),
```

Comment:
I'm not 100% all layers will have a `timeConfig`, maybe a little `?` in the mix could help
```
getters.visibleLayers.filter((layer) => layer?.timeConfig.timeEntries.length)
```

Reply:
Good point, I changed it to be on the safe side